src/mocks/handlers.ts

```diff
@@ -22,6 +22,47 @@ export const handlers = [
     ])
   }),
 
+  http.get('*/reviews*', () => {
+    return HttpResponse.json([
```

Comment:
Please move mock response data to reviewsFixture.json and import the fixture to MSW handler.